README.md

```diff
@@ -478,11 +478,11 @@ ID is "ottawa-1" for Ottawa Public Health (OPH)
 material (`saMaterial`) is `rawWW` or `sweSed` for the Public Health Agency of
 Canada (PHAC)
 
-    | ruleId | table    | mode   | key        | operator | value        | notes     |
-    |--------|----------|--------|------------|----------|--------------|-----------|
-    | 4      | samples  | select |            |          | all          |           |
-    | 5      | samples  | filter | saMaterial | =        | rawWW;sweSed |           |
-    | 6      | NA       | share  | PHAC       |          | 4;5          |           |
+    | ruleId | table   | mode   | key        | operator | value        | notes |
```

Comment:
I thought I fixed the readme, which parts are you referring to?

Reply:
1. This point should not say that it can be used for categorical type data
2. The examples should probably include one of using the `in` operator with categorical data

Reply:
done